consumer/consumererror/permanent.go

```diff
@@ -33,5 +35,5 @@ func IsPermanent(err error) bool {
 	if err == nil {
 		return false
 	}
-	return errors.As(err, &permanent{})
+	return errors.As(err, &permanent{}) || errors.As(err, &Partial{}) || errors.As(err, &TransportError{})
```

Comment:
The comment above this function should probably be updated to match this change. Right now it says: "IsPermanent checks if an error was wrapped with the NewPermanent function..."

Reply:
Good catch, fixed.